src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -54,15 +53,9 @@
  *
  */
 public abstract class CommandLineProgram implements CommandLinePluginProvider {
```

Comment:
This version of `CommandLineProgram` is corrupt -- you've undone several months of changes here. Recommend that you revert to the version in master.